rabbitmq/src/main/resources/def.json

```diff
@@ -0,0 +1,35 @@
+{
+  "vhosts": [
+    {
+      "name": "${RABBITMQ_DEFAULT_VHOST}"
```

Comment:
question: how does this `def.json` file get used? I'm having trouble tracking that down.
I do see it resembled `definition.json` - is that intentional? two minor diffs:

```
% diff def.json definitions.json 
4c4
<       "name": "${RABBITMQ_DEFAULT_VHOST}"
---
>       "name": "/"
17c17
<       "vhost": "${RABBITMQ_DEFAULT_VHOST}",
---
>       "vhost": "/",
```

Reply:
Ah that was a local test file. Shouldn't have made it into this PR. 😅